options/gridcomponents.js

```diff
@@ -139,7 +146,7 @@ class DropdownEditor {
   init(params) {
     this.gui = $('<select>' +
                   this.options.reduce((accum, curOption, i) =>
-                    DropdownEditor.getOption(params, i, browser.i18n.getMessage(curOption)) + accum
+                    accum + DropdownEditor.getOption(params, i, browser.i18n.getMessage(curOption)) + accum
```

Comment:
You have added the accum to the beginning but not removed it from the end. Now some items are repeated multiple times.

Reply:
ugh good catch. hit `c-z` too many times. fixed